### DISCUSSION TITLE
[Question][Hw1] Could you give more ciphertexts?

### DISCUSSION THREAD

**Discussion by KAIST-JongchanPark** [prosyslab-classroom/cs348-information-security#488] [Q&A]
For given ciphertexts, there are encoded with only 3~7 length words.

Could you give some more longer words, 8+ length words?

**Comment by esk223** [comment]
The document says "the word M is shorter than or equal to 8 ASCII characters (i.e., 64 bits)", but I still wonder the case that M is an 8-length word. Because we have to add an additional padding block, don't we need C0, C1, and C2?

**Comment by KAIST-JongchanPark** [comment]
Oh, i missed that statement. But as esk223 already said, there could be 8-length word M. It is helpful for to test my implementation if there are ciphertexts of 8-length words.

**Comment by Hwi-gyeom** [comment]
For the simplicity of the problem, we don't suppose 8 word situation. It will be fine if your code run in 1~7 word situation.
